src/asciidocParser.ts

```diff
@@ -172,6 +173,14 @@ export class AsciidocParser {
     }
 
     try {
+      const workspaceFolder = vscode.workspace.getWorkspaceFolder(doc.uri)
+      if (workspaceFolder !== undefined) {
+        const asciiDoctorConfig = vscode.Uri.joinPath(workspaceFolder.uri, '.asciidoctorconfig')
+        if (fs.existsSync(asciiDoctorConfig.fsPath)) {
+          const asciiDoctorConfigContent = await vscode.workspace.fs.readFile(asciiDoctorConfig)
+          text = asciiDoctorConfigContent + '\n' + text
+        }
+      }
```

Comment:
I think we should store the `asciidoctorConfig` content in the `workspaceState` to avoid reading a file every time. We should also create a `FileSystemWatcher` to update the content when the file is renamed, deleted or updated: https://code.visualstudio.com/api/references/vscode-api#workspace.createFileSystemWatcher

Reply:
Unless we detect performance issue, I would prefer to keep it to another iteration. I'm expecting that these files are usually relatively small